### ISSUE TITLE
Remove word

### ISSUE THREAD

**Issue by omerylta** [lynn/hello-wordl#132]
Love the game, but would ask to please remove the word obese as many consider it to be a slur.

**Comment by JustCallBen** [comment]
Wrong.  Clinical terms, in context, cannot be a slur.  Why do you accept fatty and not obese?
